### PR TITLE
openssl rand -base64 32

### PR DESCRIPTION
**Pull request by meehow**
It's simpler to generate encryption key on CLI

### CODE REVIEW

**Review by efectn — APPROVED** [review]

**Comment by efectn** [comment]
Don't forget to change https://github.com/gofiber/fiber/blob/master/middleware/encryptcookie/config.go#L22

And please create a PR for https://github.com/gofiber/docs repo

File: middleware/encryptcookie/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ type Config struct {
 	// Base64 encoded unique key to encode & decode cookies.
 	//
 	// Required. Key length should be 32 characters.
```
Comment (leonklingele) [inline middleware/encryptcookie/README.md:72]: Can you update this comment to mention that the key length should be 32 bytes _after_ base64-decoding:

`Required. The key should be 32 bytes of random data in base64-encoded form.`

Comment (leonklingele) [reply middleware/encryptcookie/README.md:72]: Should be done in other parts of the README and the code itself as well.

Comment (ReneWerner87) [reply middleware/encryptcookie/README.md:72]:
```suggestion
	// Required. The key should be 32 bytes of random data in base64-encoded form.
```

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**